src/components/InputField/InputField.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import {
+  BillInput,
+  BillInputContainer,
+  IconContainer,
+  StyledInput,
+  StyledInputContainer,
+  ValidationContainer,
+} from './InputField.styled';
+import { Text } from '../Text/Text';
+import colors from '../colors';
+
+const InputField = ({ title, type, icon }) => {
+  const [inputValue, setInputValue] = useState('');
+  const [error, setError] = useState('');
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleInputChange = (e) => {
+    setInputValue(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setSubmitted(true);
+
+    if (inputValue === '0' || inputValue === '') {
+      setError('Cannot be Zero');
```

Comment:
```suggestion
      setError('Cannot be Zero or Empty');
```

To match what you have in the condition statements